frontend/src/components/task/task.js

```diff
@@ -728,8 +743,17 @@ class Task extends Component {
                   open={ this.state.assignDialog }
                   onClose={ this.handleAssignDialogClose }
                   aria-labelledby='form-dialog-title'
-                  maxWidth='md'
+                  maxWidth='sm'
                 >
+                  { this.state.assignDialog ? (
+                    <Fab size='small' aria-label='close' className={ classes.closeButton } onClick={ this.handleAssignDialogClose }>
+                      <CloseIcon />
+                    </Fab>
+                  ) : null }
+                  <img
+                    src='/images/task-cover.png'
```

Comment:
This happens because frontend/public path is in .gitignore

I think the relative path always better than building an existing source but I am not sure. I will do it your way.